src/metabase/task/search_index.clj

```diff
@@ -11,22 +11,32 @@
 
 (set! *warn-on-reflection* true)
 
-;; We need each instance to initialize on start-up currently, this will need to be refined.
-(jobs/defjob ^{DisallowConcurrentExecution false
+(def ^:private recreated? (atom false))
```

Comment:
uhhh... what if you have two metabase instances?

Reply:
You'll blast away the table multiple times, which I think is OK for now. There's a comment further down about this.